<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>venPrincipal</class>
 <widget class="QMainWindow" name="venPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="panPrincipal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>951</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabClientes">
     <attribute name="title">
      <string>CLIENTES</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>951</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>331</x>
        <y>191</y>
        <width>239</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnGrabarCli">
         <property name="text">
          <string>Grabar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModificarCli">
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelCli">
         <property name="text">
          <string>Eliminar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>34</x>
        <y>60</y>
        <width>870</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblDniCli">
         <property name="text">
          <string>DNI/CIF:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtDniCli">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(243, 255, 185);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblApeCli">
         <property name="text">
          <string>Apellidos:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtApeCli">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="lblNomCli">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblEmailCli">
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtEmailCli"/>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QLineEdit" name="txtMovilCli">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblDirCli">
         <property name="text">
          <string>Dirección:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="lblProvCli">
         <property name="text">
          <string>Provincia</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="comboProvinciaCli">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLabel" name="lblMunicipoCli">
         <property name="text">
          <string>Municipio</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7" colspan="2">
        <widget class="QComboBox" name="comboMunicipioCli">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="txtDirCli">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="lblMovilCli">
         <property name="text">
          <string>Móvil:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="3">
        <widget class="QLineEdit" name="txtNomCli">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QCheckBox" name="chkHistoriaCli">
         <property name="text">
          <string>Historico</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabConstruccion">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PANEL EN CONSTRUCCION</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
